realm/realm-library/src/main/java/io/realm/internal/NativeContext.java

```diff
@@ -30,14 +30,14 @@
     private static final ReferenceQueue<NativeObject> referenceQueue = new ReferenceQueue<NativeObject>();
     private static final Thread finalizingThread = new Thread(new FinalizerRunnable(referenceQueue));
     // Dummy context which will be used by native objects which's destructors are always thread safe.
-    static final NativeContext dummyContext = new NativeContext();
+    public static final NativeContext dummyContext = new NativeContext();
```

Comment:
why public?

Reply:
Last year when discussing internal architecture we decided that to have a cleaner separation of concerns and make it easier for new people to come to the code base we should move relevant code from `internal` into `internal.objectstore` and `internal.core`. `OsObjectBuilder` lives in `internal.objectstore` now which means that some of the methods in `internal` are forced to become public.